### PR TITLE
Tweaked few explainations

### PR DESCRIPTION
**Pull request by MEME-MAN234**
I have added all the files to "foundation" and tweaked my explanations!

### CODE REVIEW

**Comment by MEME-MAN234** [comment]
@MadaZZ Thanks for the feedback! I have corrected them.

File: 1 Foundations/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+    </head>
+    <body>
+        <h1 id="date"></h1>
+        <h1 id="hello"></h1>
```
Comment (MadaZZ) [inline 1 Foundations/index.html:10]: No HTML Files needed in this repo! Read the base readme. The commits must be pure JS code. Explain how to run them in notes.

File: 1 Foundations/Change_Value.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+//<h1>This is the innerHTML place in a HTML tag</h1>
+//We can use JS to manipulate this place.Here's how-
+element_ref = document.getElementById("hello"); //Getting a HTML element reference by the element's ID
+element_ref.innerHTML = "hello"; //Changing its innerHTML
+//reference- https://www.w3schools.com/jsref/prop_html_innerhtml.asp
```
Comment (MadaZZ) [inline 1 Foundations/Change_Value.js:5]: Check out these files to see how you need to add comments for them to be more clear.
- [promise.js](https://github.com/MadaZZ/JS-Concepts/blob/master/7%20Asynchronous%20JS/1%20promise.js)
 - [modulePatterns.js](https://github.com/MadaZZ/JS-Concepts/blob/master/8%20Modules/1%20modulePatterns.js)

**Review by MadaZZ — CHANGES_REQUESTED** [review]

File: 1 Foundations/index.html (hunk 1 above)
Comment (MEME-MAN234) [reply 1 Foundations/index.html:10]: Oh, I thought it would be better. I will remove it

File: 1 Foundations/Change_Value.js (hunk 2 above)
Comment (MEME-MAN234) [reply 1 Foundations/Change_Value.js:5]: I am used to single line comments. sry

Comment (MadaZZ) [reply 1 Foundations/Change_Value.js:5]: Not a problem. But readability changes when you frame the code and comments in certain ways. 
We are all here to learn. :)

**Comment by MEME-MAN234** [comment]
@MadaZZ Please take a look at them now!

**Review by MadaZZ — APPROVED** [review]
I see everything is looking good now! 💯

**Pull request merged**